_posts/2018-11-16-scalafix-scalameta.md

```diff
@@ -0,0 +1,136 @@
+---
+category: blog
+by: Ólafur Páll Geirsson, Eugene Burmako
+title: "Build code analyzers with Scalameta and Scalafix"
+---
+
+We are excited to announce the release of
+[Scalameta v4.0.0](https://github.com/scalameta/scalameta/releases/tag/v4.0.0)
+and
+[Scalafix v0.9.0](https://github.com/scalacenter/scalafix/releases/tag/v0.8.0-RC1)
+introducing new APIs enabling more advanced source code analysis. Scalameta is a
+library to read, analyze, transform and generate Scala programs. Scalafix is a
+refactoring and linting tool.
+
+This release is the result of a close collaboration between the Scala Center and
+Twitter along with contributions from over 30 contributors in the community. Big
+thanks to everybody who made this happen!
+
```

Comment:
you might consider adding a tl;dr at this point, some bullets that say what the rest of the blog post is going to contain